lib/debezium/decimal.go

```diff
@@ -9,15 +9,20 @@ import (
 
 // EncodeDecimal is used to encode a string representation of a number to `org.apache.kafka.connect.data.Decimal`.
 func EncodeDecimal(value string, scale int) ([]byte, error) {
```

Comment:
We should probably make `scale` an `int64`.

Reply:
Really? I think if we were to change it, it should be like `uint16`

Reply:
> PostgreSQL permits the scale in a numeric type declaration to be any value in the range -1000 to 1000. However, the SQL standard requires the scale to be in the range 0 to precision. Using scales outside that range may not be portable to other database systems.

Reply:
Oh, are you mentioning this because we then don't need to convert on L17?

```go
big.NewInt(int64(scale))
```

Reply:
I just think that when it comes to mathematical operations we should avoid `int` and always try to use an integer with an explicit amount of bits (`int16`, `int32`, `int64`, etc). I would be fine with `int16` here.

Reply:
Yeah sgtm

Reply:
I'm gonna add a TODO since it'll require refactoring our decimal type too.